README.md

```diff
@@ -1,27 +1,188 @@
+| `mattermost_File_PublicLinkSalt`  |    :heavy_check_mark:    |              | Public Link Salt, generate and keep secret                                                                                                                                                             |    |    |
+| `mattermost_Email_InviteSalt`     |    :heavy_check_mark:    |              | Email Invite Salt, generate and keep secret                                                                                                                                                            |    |    |
+
+
+### Mattermost Teams
+| Name           |         Required         | Default | Description                  |    |    |
+|:---------------|:------------------------:|:--------|:-----------------------------|:---|:---|
+| `name`         |    :heavy_check_mark:    |         | Mattermost team name         |    |    |
+| `display_name` |    :heavy_check_mark:    |         | Mattermost display team name |    |    |
+| `email`        |    :heavy_check_mark:    |         | Mattermost team email        |    |    |
+| `private`      | :heavy_multiplication_x: | `false` | Is the team private          |    |    |
+
+
+### Mattermost Channels
+| Name           |         Required         | Default | Description                              |    |    |
+|:---------------|:------------------------:|:--------|:-----------------------------------------|:---|:---|
+| `name`         |    :heavy_check_mark:    |         | Mattermost channel name                  |    |    |
+| `display_name` |    :heavy_check_mark:    |         | Mattermost display name                  |    |    |
+| `team`         |    :heavy_check_mark:    |         | Mattermost team where to add the channel |    |    |
+| `private`      | :heavy_multiplication_x: | `false` | Is the channel private                   |    |    |
+
+### Mattermost Admins
+| Name        |         Required         | Default | Description           |    |    |
+|:------------|:------------------------:|:--------|:----------------------|:---|:---|
+| `username`  |    :heavy_check_mark:    |         | Username to be used   |    |    |
+| `email`     |    :heavy_check_mark:    |         | user account mail     |    |    |
+| `password`  |    :heavy_check_mark:    |         | user account password |    |    |
+| `firstname` | :heavy_multiplication_x: |         | First name            |    |    |
+| `lastname`  | :heavy_multiplication_x: |         | First name            |    |    |
+| `locale`    | :heavy_multiplication_x: |         | Language to use       |    |    |
+
+[For all config variables see the mattermost config](https://docs.mattermost.com/administration/config-settings.html#)
 
 ## Example Playbook
-
-Including an example of how to use your role (for instance, with variables passed in as parameters) is always nice for users too:
+This will follow after testing
 
 ```yml
+served_domains:
```

Comment:
Are these served_domains actually needed for mattermost? Or even used?

Reply:
they are needed for nginx